src/js/post-911-gib-status/containers/PrintPage.jsx

```diff
@@ -16,10 +15,9 @@ class PrintPage extends React.Component {
       <div className="print-status">
         <div className="print-screen">
           <img src="/img/design/logo/va-logo.png" alt="VA logo" width="300"/>
-          <h1 className="section-header">Post-9/11 GI Bill Certificate of Eligibility</h1>
+          <h1 className="section-header">Post-9/11 GI Bill Benefit Information</h1>
+          <p>Date generated: {todayFormatted}</p>
```

Comment:
I notice that not all the mocks have the "Date generated" message  (e.g., https://marvelapp.com/8d6igd9/screen/29016777 is missing it), but I'm guessing it should show up for at least all of the cases where the person is enrolled. @phiden can you confirm or correct?